src/components/UserProfile/UserProfile.jsx

```diff
@@ -527,7 +587,7 @@ const UserProfile = props => {
                 userPermissions={userPermissions}
                 canEdit={canEdit}
               />
-              <BlueSquareLayout
+              {/* <BlueSquareLayout
```

Comment:
Don't forget to uncomment BlueSquareLayout. Otherwise blue squares will not display once your PR gets merged.